bitnami/kong/templates/dep-ds.yaml

```diff
@@ -365,15 +373,20 @@ spec:
         {{- if .Values.kong.initScriptsCM }}
         - name: custom-init-scripts-cm
           configMap:
-            name: {{ .Values.kong.initScriptsCM }}
+            name: {{ tpl .Values.kong.initScriptsCM $ }}
```

Comment:
Same comment about `tpl`, I believe you should use our `common.tplvalues.render` function